src/appmixer/cloudflare/service.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "appmixer.cloudflare",
+    "label": "Salt CloudFlare support",
+    "category": "applications",
+    "description": "CloudFlare is an American company that provides content delivery network services",
```

Comment:
Change to "CloudFlare provides content delivery network services, cloud cybersecurity, DDoS mitigation, wide area network services, reverse proxies, Domain Name Service, and ICANN-accredited domain registration services."